### PR TITLE
activity: use lastGiven header

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Fix https://github.com/nextcloud/android/issues/4272

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### CODE REVIEW

**Comment by tobiasKaminsky** [comment]
@Boballen55 thanks for testing, this was very important!
I indeed added a limit of 5 for fetching activities, instead of default of 50.

I changed it, so it will be fast again.

File: src/main/java/com/owncloud/android/ui/activities/ActivitiesActivity.java [hunk 1]
```diff
@@ -125,8 +125,8 @@ protected void onCreate(Bundle savedInstanceState) {
         swipeListRefreshLayout.setOnRefreshListener(() -> {
             // We set the nextPageUrl variable to null here since when manually refreshing
             // activities data we want to clear the list and reset the pagination.
-            nextPageUrl = null;
-            mActionListener.loadActivities(nextPageUrl);
+            lastGiven = -1;
```
Comment (AndyScherzinger) [inline src/main/java/com/owncloud/android/ui/activities/ActivitiesActivity.java]: It seems to me "-1" is a magic number and should be defined as a constant.

Comment (tobiasKaminsky) [reply src/main/java/com/owncloud/android/ui/activities/ActivitiesActivity.java]: Nice catch, this makes it indeed more readable

**Pull request merged**